_posts/2024-08-02-gsoc2024_talagayev.md

```diff
@@ -0,0 +1,193 @@
+
+```python
+import MDAnalysis as mda
+import mdonatello
+from mdonatello import MoleculeVisualizer
+
+u = mda.Universe("input.pdb")
+ag = u.select_atoms("resname UNK")
+visualizer = MoleculeVisualizer(ag, show_atom_indices=False, width=-1, height=-1)
+```
+
+This would lead you to obtain the following display, where you could then select the molecule and checkboxes:
+
+<img
+src="/public/images/mdonatello_display.png"
+title="MDonatello Display" alt="Example of MDonatello Display"
+style="float: center; " />
+
+## Lessons Learned During GSoC
+
+Here I would highlight some lessons that I learned during GSoC2024:
+
+- Have an Idea of the overall structure of the code
+- Discuss the ideas with the mentors to see their opinion
+- Don't be afraid to ask questions
+- Try to work on different aspects of the code (in my case I was almost always only focusing on feature addition and needed to get reminded that there are other parts)
+- Try to learn as much from the mentors and apply it in your code
+- Getting a meeting with all mentors that have different time zones is sometimes tricky :)
+
+## Conclusion &The Future
+
+It was a fun project and I liked working on it, but the work doesn't stop here. As I mentioned some aspects of the code can be improved and things that can be
+done, so I will continue to work on the code after GSoC 2024, here are again the short highlights that I already mentioned that I would work on:
+
+- Code structure improvement
+- Figure Generation and SMART Patterns need to be improved
+- Heatmap needs to be created for partial charges instead of atom highlighting
+- Adding the package to `MDAKits` and `conda-forge`
+- New Features can be added
+
+## Acknowledgements
+
+I would like to thank [MDAnalysis](https://www.mdanalysis.org/) for giving me the opportunity to work on this project. The application process was very nice and detailed,
+I liked it that during the application process. I was able to get to know the MDAnalysis code more and contribute to the code and get to know some people from the community
+during this time.
+
+I would like to thank Oliver Beckstein ([@orbeckst](https://github.com/orbeckst)) and Jenna M Swarthout Goddard([@jennaswa](https://github.com/jennaswa)) for the insights and help with the organization and structure during the application, I am glad that I was able to contribute to MDAnalysis with this project.
+
+I would also like to thank my mentors Cédric Bouysset ([@cbouy](https://github.com/cbouy)), Richard Gowers ([@richardjgowers](https://github.com/richardjgowers)) and Yuxuan Zhuang ([@yuxuanzhuang](https://github.com/yuxuanzhuang)), who helped me a lot during the project, with their helpful and insightful mentoring. I am glad, that I was able to learn from you and it helped me to improve
+my coding skills. Also, shotout to Hugo MacDermott-Opeskin ([@hmacdope](https://github.com/hmacdope)) for his help with the PR requests during the application process.
```

Comment:
```suggestion
my coding skills. Also, shoutout to Hugo MacDermott-Opeskin ([@hmacdope](https://github.com/hmacdope)) for his help with the PR requests during the application process.
```

Reply:
Damn, sorry for the typo errors, I checked it beforehand and thought that there were no typo errors left :crying_cat_face: